commit 36195a3781c682df0af907c374a9a5b8cdb80b84 (refs/changes/08/3508/3)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-02-28T00:49:07+00:00 (1 year, 7 months ago)
    
    TRITON-191 disable support for passing brand to cloudapi
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/docs/index.md b/docs/index.md
index 42df2cf..9716920 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -908,6 +908,12 @@ The set of supported *API versions* is given in the ping endpoint:
 
 The section describes API changes in CloudAPI versions.
 
+
+## 8.7.0
+
+- CreateMachine no longer accepts the `brand` field for specifying the brand
+  of the instance to create.
+
 ## 8.6.0
 
 - CreateMachine now accepts the `brand` field for specifying the brand of the
@@ -4357,7 +4363,6 @@ be changed later within the instance, if desired.
 
 **Field** | **Type** | **Description**
 --------- | -------- | ---------------
-brand     | String   | Optionally specify brand for this instance. Currently only useful to distinguish between bhyve and kvm in datacenters that support both. Possible values: "bhyve", "joyent", "kvm".
 name      | String   | Friendly name for this instance; default is the first 8 characters of the machine id. If the name includes the string {{shortId}}, any instances of that tag within the name will be replaced by the first 8 characters of the machine id.
 package   | String   | Id of the package to use on provisioning, obtained from [ListPackages](#ListPackages)
 image     | String   | The image UUID (the "id" field in [ListImages](#ListImages))
diff --git a/lib/machines.js b/lib/machines.js
index efdc628..f2e3451 100644
--- a/lib/machines.js
+++ b/lib/machines.js
@@ -293,10 +293,6 @@ function getListOptions(req) {
                                         req.params.type);
     }
 
-    if (req.params.brand) {
-        opts.brand = req.params.brand;
-    }
-
     if (req.params.image) {
         opts.image_uuid = req.params.image;
     }
diff --git a/package.json b/package.json
index 3582e97..2f349c3 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "cloudapi",
     "description": "SmartDataCenter CloudAPI",
-    "version": "8.6.1",
+    "version": "8.7.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "engines": {
